gamedata/myz/myz-monsters-catalog.en.csv

```diff
@@ -1,5 +1,5 @@
 id;strength;agility;wits;empathy;speed;hp;armor;skills;special;source;attacks
-mmyz-humanoid-amnesiac;2;2;4;2;1;1;0;Comprehend 2|Heal 1;;myz;{unarmed}
+mmyz-humanoid-amnesiac;2;2;4;2;1;1;0;Comprehend 2|Heal 1;;myz;{wmyz-unarmed}
```

Comment:
This change must be double-checked

Reply:
{unarmed} would just say "Unarmed". And {wmyz-unarmed} would cover that for attacks, wouldn't it?

Reply:
You are right, but the syntax `{something}` also returns a default weapon, which is... an unarmed attack.. but with range!

So yes, your correction is better and fixes an issue I didn't see.

See: https://github.com/Stefouch/sebedius-myz-discord-bot/blob/fae188c804cbc79a493fa3a3691506367f974b47/yearzero/YZObject.js#L343